stdlib/public/core/Span/RawSpan.swift

```diff
@@ -12,28 +12,32 @@
 
 // A RawSpan represents a span of initialized memory
 // of unspecified type.
+@_disallowFeatureSuppression(NonescapableTypes)
 @frozen
-public struct RawSpan: Copyable, ~Escapable {
+public struct RawSpan: ~Escapable, Copyable, BitwiseCopyable {
   @usableFromInline let _pointer: UnsafeRawPointer?
 
   @usableFromInline @inline(__always)
   var _start: UnsafeRawPointer { _pointer.unsafelyUnwrapped }
 
   @usableFromInline let _count: Int
 
-  @_alwaysEmitIntoClient
-  internal init(
-    _unchecked start: UnsafeRawPointer?,
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @usableFromInline @inline(__always)
+  @lifetime(immortal)
```

Comment:
Should be `@lifetime(pointer)`. The resulting `Span` depends on the validity of the pointer.